templates/sample-app/res/src/apex/force-app/main/default/classes/DataDeleter.cls

```diff
@@ -26,19 +26,19 @@
 
 public class DataDeleter 
 {
-	public static void deleteData()
-	{
-		Id userId = UserInfo.getUserId();
-		delete [SELECT Id FROM Vehicle__c];
-		delete [SELECT Id FROM VehicleType__c];
+    public static void deleteData()
+    {
+        Id userId = UserInfo.getUserId();
+        delete [SELECT Id FROM Vehicle__c];
```

Comment:
Hmm, I'm surprised this gets past the Limits, but I guess it's unchanged!

Reply:
This doesn't get past the limits if the multiple Delivery plans have been created. Leaving as I haven't changed this.